src/frontends/tensorflow_lite/src/op/gelu.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2018-2023 Intel Corporation
+// Copyright (C) 2024 Intel Corporation
```

Comment:
We should squash this 2nd commit to the 1st one, maybe just split the layer test if you want to.